src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2381,6 +2382,17 @@ public function setIncrementing($value)
     }
 
     /**
+     * Transform each attribute in the model using a callback.
+     *
+     * @param  callable  $callback
+     * @return \Illuminate\Support\Fluent
+     */
+    public function transform(callable $callback)
+    {
+        return (new Fluent($this->toArray()))->transform($callback);
```

Comment:
Which is why it's a simple transformer.

Fractal does more with formatting the input via transformer and then formatting the output with serializer. `Fluent` and `Collection` (which already support `transform()` method) offer simple json serializer out of the box.